content/en/docs/kubernetes/collector/components.md

```diff
@@ -661,3 +663,90 @@ subjects:
     name: otel-collector-opentelemetry-collector
     namespace: default
 ```
+
+## Host Metrics Receiver
+
+| Deployment Pattern   | Usable                                                        |
+| -------------------- | ------------------------------------------------------------- |
+| DaemonSet (Agent)    | Preferred                                                     |
+| Deployment (Gateway) | but will only collect metrics from the node it is deployed on |
+| Sidecar              | No                                                            |
+
+The Host Metrics Receiver collects metrics from a host using a variety of
+scrappers. There is some overlap with the
+[Kubeletstats Receiver](#kubeletstats-receiver) so if you decide to use both it
+may be worth it to disable these duplicate metrics.
```

Comment:
```suggestion
The Host Metrics Receiver collects metrics from a host using a variety of
scrapers. There is some overlap with the
[Kubeletstats Receiver](#kubeletstats-receiver) so if you decide to use both,
it may be worth it to disable these duplicate metrics.
```